model_compression_toolkit/core/common/back2framework/base_model_builder.py

```diff
@@ -0,0 +1,34 @@
+from typing import Any, Tuple
+
+from model_compression_toolkit.core import common
+from model_compression_toolkit.core.common.user_info import UserInformation
+
+
+class BaseModelBuilder:
+    """
+    Base class for model builder.
+    """
+
+    def __init__(self,
+                 graph: common.Graph,
+                 append2output=None,
+                 return_float_outputs: bool = False):
```

Comment:
why fw_info is not part of BaseModelBuilder?

Reply:
Because it's related to a specific framework.